docs/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
```

Comment:
It seems like the wrong file was copied. The contents of this file should be equal to https://github.com/scylladb/sphinx-scylladb-theme/blob/master/docs/_utils/pyproject_template.toml and not https://github.com/scylladb/sphinx-scylladb-theme/blob/master/pyproject.toml.

To fix the issue, delete the file and...
> Create the file docs/pyproject.toml under the new docs folder. Copy the contents from the [pyproject.toml template](https://github.com/scylladb/sphinx-scylladb-theme/blob/master/docs/_utils/pyproject_template.toml).